public/achievementList.php

```diff
@@ -196,13 +205,19 @@
             echo gameAvatar($achEntry, label: false);
             echo "</td>";
 
-            echo "<td>";
-            echo "<span class='smalldate'>" . getNiceDate(strtotime($achDateCreated)) . "</span>";
-            echo "</td>";
+            if (!$isOwnEarnedAchievementsList) {
+                echo "<td>";
+                echo "<span class='smalldate'>" . getNiceDate(strtotime($achDateCreated)) . "</span>";
+                echo "</td>";
 
-            if (!$mobileBrowser) {
+                if (!$mobileBrowser) {
+                    echo "<td>";
+                    echo "<span class='smalldate'>" . getNiceDate(strtotime($achDateModified)) . "</span>";
+                    echo "</td>";
+                }
+            } else {
                 echo "<td>";
-                echo "<span class='smalldate'>" . getNiceDate(strtotime($achDateModified)) . "</span>";
+                echo "<span class='smalldate'>" . getNiceDate(strtotime($achAwardedDate)) . "</span>";
```

Comment:
This generates `01 Jan 1970, 00:00` if `$achEntry['AwardedDate']` is null (Achievements in Core Set, Awarded ascending).

Reply:
Yep! I didn't realize `AwardedDate` is actually nullable. Now these values will display "Unknown".